solutions/usaco-940.mdx

```diff
@@ -87,6 +87,61 @@ public class Factory {
 ```
 
 </JavaSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+void setIO(string name=""){ // FastIO See General -> Input/Output
+    ios_base::sync_with_stdio(0); cin.tie(0);
+    if((int)name.size() > 0){
+        freopen((name+".in").c_str(), "r", stdin);
+        freopen((name+".out").c_str(), "w", stdout);
+    }
+}
+
+int main(){
+    setIO("factory");
+    int N;
+    cin >> N;
+    int outgoing[N] = {0};
```

Comment:
arrays are not meant to be dynamically sized, use vectors instead

Reply:
Done